### PR TITLE
// classic: design improvements

### PR DESCRIPTION
**Pull request by nihco2**

### CODE REVIEW

File: themes/classic/templates/catalog/product-miniature.tpl [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <div class="product-description">
       {block name='product_name'}
-        <h1 class="h3 product-title" itemprop="name"><a href="{$product.url}">{$product.name}</a></h1>
+        <h1 class="h3 product-title" itemprop="name"><a href="{$product.url}">{$product.name|truncate:30:'...'}</a></h1>
```
Comment (djfm) [inline themes/classic/templates/catalog/product-miniature.tpl]: Would `text-overflow: ellipsis` work here? (I know it only works in a very limited subset of cases and I'm usually disappointed)

Comment (nihco2) [inline themes/classic/templates/catalog/product-miniature.tpl]: It seems it don't works here

File: themes/classic/templates/catalog/products.tpl [hunk 2]
```diff
@@ -33,7 +33,7 @@
       <div class="products row">
         {foreach from=$products item="product"}
           {block name='product_miniature'}
-            {include file='catalog/product-miniature.tpl' product=$product columns=4}
+            {include file='catalog/product-miniature.tpl' product=$product columns=3}
```
Comment (djfm) [inline themes/classic/templates/catalog/products.tpl]: `columns` argument can be removed, right?

File: themes/classic/modules/homefeatured/views/templates/hook/homefeatured.tpl [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <h1 class="h3 products-section-title text-uppercase ">{l s='Popular Products'}</h1>
   <div class="products">
     {foreach from=$products item="product"}
-      {include file="catalog/product-miniature.tpl" product=$product columns=3}
+      {include file="catalog/product-miniature.tpl" product=$product columns=4}
```
Comment (djfm) [inline themes/classic/modules/homefeatured/views/templates/hook/homefeatured.tpl]: `columns` argument can be removed, right?

File: themes/classic/templates/catalog/products.tpl (hunk 2 above)
Comment (nihco2) [inline themes/classic/templates/catalog/products.tpl]: yep

File: themes/classic/templates/catalog/product-miniature.tpl (hunk 1 above)
Comment (djfm) [inline themes/classic/templates/catalog/product-miniature.tpl]: Too bad :(

**Pull request merged**